tests/Unit/Application/User/Command/SendConfirmationLinkCommandHandlerTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Unit\Application\User\Command;
+
+use App\Application\DateUtilsInterface;
+use App\Application\Mail\User\ConfirmAccountMailInterface;
+use App\Application\User\Command\SendConfirmationLinkCommand;
+use App\Application\User\Command\SendConfirmationLinkCommandHandler;
+use App\Domain\User\User;
+use PHPUnit\Framework\TestCase;
+
+final class SendConfirmationLinkCommandHandlerTest extends TestCase
+{
+    public function testSendUserAccountVerification(): void
+    {
+        $now = new \DateTimeImmutable('2023-08-21');
+        $user = $this->createMock(User::class);
+        $user
+            ->expects(self::once())
+            ->method('getEmail')
+            ->willReturn('mathieu@fairness.coop');
+
+        $dateUtils = $this->createMock(DateUtilsInterface::class);
+        $dateUtils
+            ->expects(self::once())
+            ->method('getNow')
+            ->willReturn($now);
+
+        $confirmAccountMail = $this->createMock(ConfirmAccountMailInterface::class);
+        $confirmAccountMail
+            ->expects(self::once())
+            ->method('send')
+            ->with('mathieu@fairness.coop', new \DateTimeImmutable('2023-08-31'));
+
+        $handler = new SendConfirmationLinkCommandHandler(
+            $confirmAccountMail,
+            $dateUtils,
+        );
+
+        $handler->__invoke(new SendConfirmationLinkCommand($user));
```

Comment:
J'utilise plutôt cette notation
```suggestion
        ($handler)(new SendConfirmationLinkCommand($user));
```
mais c'est un feeling perso, je ne sais pas s'il y a une bonne pratique en la matière

Reply:
Ca me va aussi